### PR TITLE
映射文档 No.24

### PR DESCRIPTION
**Pull request by ccsuzzh**
分组 ID 24
- #5691

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.var.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```python
+paddle.Tensor.var(axis=None, unbiased=True, keepdim=False, name=None)
+```
+
+两者功能一致且参数用法一致，仅参数名不同，具体如下：
+### 参数映射
+
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| <font color='red'> dim </font> | <font color='red'> axis </font> | 指定对 Tensor 进行计算的轴 ，仅参数名不一致。   |
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.var.md]: 参数名相同的参数也补充一下吧~ 例如 `unbiase`, `keepdim`
| unbiased | unbiased | 表示是否使用无偏估计来计算方差。 |
| keepdim | keepdim |  表示是否保留计算后的维度。 |

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.Event.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+torch.cuda.Event(enable_timing=False, blocking=False, interprocess=False)
+```
+
+### [paddle.device.cuda.Event](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/var_cn.html#varhttps://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/device/cuda/Event_cn.html#event)
+
+```python
+paddle.device.cuda.Event(enable_timing=False, blocking=False, interprocess=False)
+```
+
+功能一致，参数完全一致。
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.Event.md]: 这里虽然参数一致，也辛苦写一下参数映射部分吧~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.current_stream.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+torch.cuda.current_stream(device=None)
+```
+
+### [paddle.device.cuda.current_stream](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/device/cuda/current_stream_cn.html)
+
+```python
+paddle.device.cuda.current_stream(device=None)
+```
+
+功能一致，参数完全一致。
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.current_stream.md]: 这里虽然参数一致，也辛苦写一下参数映射部分吧~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_capability.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+torch.cuda.get_device_capability(device=None)
+```
+
+### [paddle.device.cuda.get_device_capability](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/device/cuda/get_device_capability_cn.html)
+
+```python
+paddle.device.cuda.get_device_capability(device=None)
+```
+
+功能一致，参数完全一致。
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_capability.md]: 这里虽然参数一致，也辛苦写一下参数映射部分吧~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_properties.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+torch.cuda.get_device_properties(device)
+```
+
+### [paddle.device.cuda.get_device_properties](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/device/cuda/get_device_properties_cn.html)
+
+```python
+paddle.device.cuda.get_device_properties(device)
+```
+
+功能一致，参数完全一致。
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_properties.md]: 这里虽然参数一致，也辛苦写一下参数映射部分吧~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.set_device.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+torch.cuda.set_device(device)
+```
+
+### [paddle.device.set_device](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/device/set_device_cn.html)
+
+```python
+paddle.device.set_device(device)
+```
+
+功能一致，参数完全一致。
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.set_device.md]: 这里虽然参数一致，也辛苦写一下参数映射部分吧~

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+torch.Tensor.where(condition, y)
+```
+
+### [paddle.Tensor.where](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/Tensor_cn.html#where-y-name-none)
+
+```python
+paddle.Tensor.where(y, name=None)
+```
+
+两者功能一致且参数用法一致。
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: pytorch 相比 paddle 多condition 参数，所以分类应该是`torch 参数更多`，关于这个参数，是需要`转写`的。pytorch必须传入Tensor类型的condition，当condition为true时选择`x`中元素，反之选择y中元素。而paddle是直接将`x`作为condition，如果x元素小于0，则选择y中的元素。
```python
import torch

a = torch.tensor([0, 1, 2])
b = torch.tensor([2, 3, 0])
c = a.where(a>0, b)
print(c)
```

```python
import paddle
a = paddle.to_tensor([0, 1, 2])
b = paddle.to_tensor([2, 3, 0])
c = a.where(b)
print(c)
```
paddle的此方法有点问题，明天我再确认一下

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.var.md (hunk 1 above)
Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.var.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.Event.md (hunk 2 above)
Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.Event.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.current_stream.md (hunk 3 above)
Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.current_stream.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_capability.md (hunk 4 above)
Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_capability.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_properties.md (hunk 5 above)
Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.get_device_properties.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.set_device.md (hunk 6 above)
Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/cuda/torch.cuda.set_device.md]: done

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md (hunk 7 above)
Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 请问一下，这个问题现在确定了吗？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+a = torch.tensor([0, 1, 2])
+b = torch.tensor([2, 3, 0])
+c = a.where(a>0, b)
+print(c)
+
+# paddle 写法
+import paddle
+a = paddle.to_tensor([0, 1, 2])
+b = paddle.to_tensor([2, 3, 0])
+c = a.where(b)
```
Comment (Fancy-hjyp) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 这里是官网文档的问题，正确的使用方式应该是，辛苦转写改成这样吧~
```python
import paddle
a = paddle.to_tensor([3, 2, 0])
b = paddle.to_tensor([0, 4, 5])
c = (a > 0).where(a, b)
print(c)
```

Comment (Fancy-hjyp) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 转写示例不需要写 `import paddle` `print`等语句，只写核心代码即可

Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 如果是这样，那参数是不是就与pytorch的参数是一致的了，应该属于参数完全一致的分类？

Comment (Fancy-hjyp) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 还是不一样的，pytorch 是`c = a.where(a>0, b)`，condition是`a>0`，作为参数传递；paddle 是 `c = (a > 0).where(a, b)`，condition在外面，这个先归为`参数用法不一致`吧，添加转写示例。

Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 好的。

**Review by Fancy-hjyp — APPROVED** [review]
LGTM

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+```python
+paddle.Tensor.where(x, y, name=None)
+```
+
+两者功能一致，仅参数用法不一致，具体如下：
+### 参数差异
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| condition        | x            | 在 pytorch 中 condition 为判断条件，而在 paddle 中 condition 需要转写， 当 condition 为 true 时，选择 x 中元素。                                     |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 这两个是对不上的，可以不用放一行。而且这段话写的有点绕。建议这里用paddle.where来实现：paddle.where(condition, self, y)

Comment (ccsuzzh) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.where.md]: 好的。意思是用paddle.where来转写吗？

**Review by zhwesky2010 — APPROVED** [review]
